apps/services/party-letter-registry-api/migrations/20210514135724-party-letter-registry.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    const transaction = await queryInterface.sequelize.transaction()
+
+    // try is here to make sure this safely fails if index creation fails
+    try {
+      await queryInterface.createTable('party_letter_registry', {
+        id: {
+          type: Sequelize.INTEGER,
+          autoIncrement: true,
+          primaryKey: true,
+        },
+        party_letter: {
+          type: Sequelize.CHAR(2),
+          allowNull: false,
+          unique: true,
+        },
+        party_name: {
+          type: Sequelize.STRING,
+          allowNull: false,
+        },
+        owner: {
+          type: Sequelize.STRING,
+          allowNull: false,
+          unique: true,
+        },
+        managers: {
+          type: Sequelize.ARRAY(Sequelize.STRING),
+          allowNull: false,
+        },
+        created: {
+          type: Sequelize.DATE,
+          allowNull: false,
+        },
+        modified: {
+          type: Sequelize.DATE,
+          allowNull: false,
+        },
```

Comment:
Should this be based on election terms, or should I be able to use some application to "edit" this data for future elections?

Reply:
We plan for this to be a database containing the current state of party letters.
We consider this a source of truth when managing applications for party letter related activities